ote/src/clj/ote/transit_changes/detection.clj

```diff
@@ -12,14 +12,11 @@
             [ote.util.collections :refer [map-by count-matching]]
             [ote.util.functor :refer [fmap]]
             [ote.db.tx :as tx]
-            [ote.transit-changes.change-history :as change-history])
+            [ote.transit-changes.change-history :as change-history]
+            [ote.config.transit-changes-config :as config-transit-chg])
```

Comment:
Should this be imported with the same `:as` in every namespace it's used in? Is called `config-tc` in gtfs.clj.

Reply:
Renamed.